x/shield/keeper/params.go

```diff
@@ -40,3 +40,14 @@ func (k Keeper) GetShieldStakingRate(ctx sdk.Context) (rate sdk.Dec) {
 func (k Keeper) SetShieldStakingRate(ctx sdk.Context, rate sdk.Dec) {
 	k.paramSpace.Set(ctx, types.ParamStoreKeyStakingShieldRate, &rate)
 }
+
+// GetDistributionParams returns distribution parameters.
+func (k Keeper) GetDistributionParams(ctx sdk.Context) (distrParams types.) {
```

Comment:
I guess the `distrParams types.` in function signature should be replaced with `rate sdk.Dec`?